src/command_modules/azure-cli-locationbasedservices/azure/cli/command_modules/locationbasedservices/custom.py

```diff
@@ -0,0 +1,134 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.log import get_logger
+from knack.prompting import prompt_y_n
+from knack.util import CLIError
+
+from azure.mgmt.locationbasedservices.models import (
+    LocationBasedServicesAccountCreateParameters,
+    Sku)
+
+ACCOUNT_LOCATION = 'global'
+
+logger = get_logger(__name__)
+
+
+# pylint: disable=line-too-long
+def get_account(client, resource_group_name, account_name):
+    """Get a Location Based Services Account.
+
+    :param resource_group_name: The name of the Azure Resource Group.
+    :type resource_group_name: str
+    :param account_name: The name of the Location Based Services Account.
+    :type account_name: str
+    :return: LocationBasedServicesAccount
+    :rtype:
+     ~azure.mgmt.locationbasedservices.models.LocationBasedServicesAccount
+    :raises:
+     :class:`ErrorException<azure.mgmt.locationbasedservices.models.ErrorException>`
+     or :class:`ErrorException<knack.util.CLIError>`
+    """
```

Comment:
There are a few ways of supplying help text to the CLI. The docstring for a custom method is the only method that is actively discouraged.  It would make more sense to put this in the help.py file so it is easier for people to find if they need to edit it (a docs feature) and because that system has more capabilities.  If you just want to reuse help text across commands, then use the `help` kwarg in params.py.